packages/grafana-ui/src/components/BigValue/BigValue.tsx

```diff
@@ -66,8 +66,6 @@ export interface Props extends Themeable2 {
   textMode?: BigValueTextMode;
   /** If true disables the tooltip */
   hasLinks?: boolean;
-  /** The orientation of the parent container */
```

Comment:
Could you please mark this as deprecated instead of removing it?

Reply:
@academo not sure we need to, it's not needed, does nothing. And I doubt no plugin has ever used this in the short time after it was added. in fact I cannot find a single plugin that imports BigValue when scanning all the plugin imports in big query. 

And the only effect is that when they update is the property will be removed (no runtime errors), but since it was optional and basically did nothing I highly doubt anyone bothered to set it